shared/video-sdk/develop/ensure-channel-quality/project-implementation/windows.mdx

```diff
@@ -2,7 +2,7 @@
 
 ### Implement the user interface
 
-This section guides you through the necessary UI changes in the _Get Started_ project interface to implement call quality features. 
+This section guides you through the necessary UI changes in the _<Vg k="GET_STARTED"/>_ project interface to implement call quality features. 
```

Comment:
There are a few instances where this title is italicized, and some where it is not. Would be nice to have it unified.

Reply:
Removed the italics.